### PR TITLE
Extra S3 deb test via curl PUT

### PR DESCRIPTION
**Pull request by ChGen**
Extra S3 deb test via curl PUT.

### CODE REVIEW

**Comment by ChGen** [comment]
@dgarus , please review

File: artipie-main/src/test/java/com/artipie/debian/DebianS3ITCase.java [hunk 1]
```diff
@@ -97,6 +97,48 @@ void setUp() throws IOException {
         );
     }
 
+    @ParameterizedTest
+    @CsvSource({
+        "8080,my-debian,9000"
+    })
+    void curlPutWorks(final String port, final String repo, final String s3port) throws Exception {
```
Comment (dgarus) [inline artipie-main/src/test/java/com/artipie/debian/DebianS3ITCase.java:104]: Why is this test `@ParameterizedTest`?

Comment (ChGen) [reply artipie-main/src/test/java/com/artipie/debian/DebianS3ITCase.java:104]: @dgarus , it makes code in tests more concise and + allows to test several variants, in perspective.

Comment (dgarus) [reply artipie-main/src/test/java/com/artipie/debian/DebianS3ITCase.java:104]: As I can see, this test is executed with just one `csv` line, so you can define `port`, `repo`, `s3port` as local variables. Isn't it?

Comment (ChGen) [reply artipie-main/src/test/java/com/artipie/debian/DebianS3ITCase.java:104]: @dgarus , ok.

**Comment by dgarus** [comment]
@ChGen Clould you please remove all `@checkstyle` lines in the project?

**Comment by ChGen** [comment]
@dgarus , ok, I will remove `@checkstyle` for artipie-main in separate PR.

**Review by dgarus — DISMISSED** [review]

**Comment by ChGen** [comment]
@dgarus please review

**Review by dgarus — APPROVED** [review]

**Pull request merged**